app/views/check_your_answers.scala.html

```diff
@@ -47,7 +47,13 @@ <h2 class="govuk-heading-l">@messages("checkYourAnswers.declaration.subheading")
             <ul class="govuk-list govuk-list--bullet">
                 <li>@messages("checkYourAnswers.declaration.listItem1")</li>
                 <li>@messages("checkYourAnswers.declaration.listItem2")</li>
-                <li>@Html(messages("checkYourAnswers.declaration.listItem3", messages("checkYourAnswers.declaration.email"), messages("checkYourAnswers.declaration.email")))</li>
+                <li>@messages("checkYourAnswers.declaration.listItem3")</li>
+                @if(messages.lang.code == "cy"){
```

Comment:
I'm not sure I understand why we have this logic here? The language switch is dealt with by the wrapper [here](https://github.com/hmrc/binding-tariff-trader-frontend/blob/05c07d9e21b838c982ec8ddec9db2f235c96873f/app/views/GovukLayoutWrapper.scala.html#L67) so there shouldn't be a need to add this logic in the views. Can we talk through this together?